### PR TITLE
support tls for SkyWalking client

### PR DESCRIPTION
**Pull request by dmsolr**

### CODE REVIEW

File: pkg/exporter/skywalking.go [hunk 1]
```diff
@@ -67,7 +78,39 @@ func (exporter *SkyWalking) Init(ctx context.Context) error {
 		return err
 	}
 
-	conn, err := grpc.Dial(config.Address, grpc.WithInsecure())
+	var dialOption grpc.DialOption
+	if config.EnableTLS {
+		if isFileExisted(config.ClientCertPath) && isFileExisted(config.ClientKeyPath) {
+			clientCert, err := tls.LoadX509KeyPair(config.ClientCertPath, config.ClientKeyPath)
+			if err != nil {
+				log.Fatalf("Failed to load client certificate and key. %s.", err)
```
Comment (kezhenxu94) [inline pkg/exporter/skywalking.go:86]: return the error instead of failing here

File: pkg/exporter/skywalking.go [hunk 2]
```diff
@@ -67,7 +78,39 @@ func (exporter *SkyWalking) Init(ctx context.Context) error {
...
+			}
+			trustedCert, err := ioutil.ReadFile(config.TrustedCertPath)
+			if err != nil {
+				log.Fatalf("Failed to load trusted certificate. %s.", err)
```
Comment (kezhenxu94) [inline pkg/exporter/skywalking.go:90]: return the error instead of failing here

File: pkg/exporter/skywalking.go [hunk 3]
```diff
@@ -67,7 +78,39 @@ func (exporter *SkyWalking) Init(ctx context.Context) error {
...
+			}
+			certPool := x509.NewCertPool()
+			if !certPool.AppendCertsFromPEM(trustedCert) {
+				log.Fatalf("Failed to append trusted certificate to certificate pool. %s.", err)
```
Comment (kezhenxu94) [inline pkg/exporter/skywalking.go:94]: return the error instead of failing here

**Review by kezhenxu94 — COMMENTED** [review]
Some nits, otherwise LGTM

**Comment by kezhenxu94** [comment]
Fix the CI please

**Review by kezhenxu94 — APPROVED** [review]

**Comment by dmsolr** [comment]
Thank you @kezhenxu94 , this is my first PR in GoLang, thanks. :)

**Pull request merged**